android/build.gradle

```diff
@@ -41,5 +41,5 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
-    implementation 'com.facebook.android:facebook-android-sdk:12.+'
+    implementation 'com.facebook.android:facebook-android-sdk:12.0.0'
```

Comment:
This would be a locked in downgrade of the SDK to enforce a lower version locked to be exactly `12.0.0` https://docs.gradle.org/current/userguide/single_versions.html
 
Please help me understand what issues that the `12.+` declaration caused?

Reply:
mavenCentral uses a different schema than jcenter, and doesn't seem to recognize the '+' pattern. Trying to build the repo using '12.+' throws the error below (I'm using an older version of this package in the screenshot below that's linked to 9.+)

![fb_error](https://user-images.githubusercontent.com/6893675/145918780-35b197a6-47ef-4ccd-b7a9-0323756c81c2.PNG)

Reply:
I think it's better to be explicit than implicit. 

According to the documentation, it seems like `[12.0,13.0)` is the most unambiguous version declaration that is semantically the same as `12.+`

![image](https://user-images.githubusercontent.com/2051607/145921575-44db0e48-69c6-4d00-8522-99689ab60371.png)

Reply:
Also doesnt work for some reason

![image](https://user-images.githubusercontent.com/6893675/145923839-ee4202fe-7824-4441-a042-727a6a1e7120.png)